integration-cli/docker_cli_stats_test.go

```diff
@@ -118,16 +118,17 @@ func (s *DockerSuite) TestStatsAllNewContainersAdded(c *check.C) {
 			switch {
 			case matchID.MatchString(scanner.Text()):
 				close(addedChan)
+				return
 			}
 		}
 	}()
 
-	out, _ := dockerCmd(c, "run", "-d", "busybox", "top")
+	out, _ := runSleepingContainer(c, "-d")
 	c.Assert(waitRun(strings.TrimSpace(out)), check.IsNil)
 	id <- strings.TrimSpace(out)[:12]
 
 	select {
-	case <-time.After(10 * time.Second):
+	case <-time.After(30 * time.Second):
```

Comment:
well, this was @ 5 seconds on my branch... 10 seems like it should be ok, but...